src/styles/main.scss

```diff
@@ -1,14 +1,14 @@
-html {
-  font-family: Roboto, sans-serif;
-}
+// utils import
```

Comment:
are you really need this comments?

Reply:
I think all these comments are ok. They help me and others distinguish different parts of my abstract files

Reply:
These comments are ok, but I agree that there is no need to write comments everywhere, 'cause your code should be understandable without it. For such purpose - up to you, you may leave it as it is now